plugins/modules/keycloak_realm_key.py

```diff
@@ -103,7 +103,7 @@
                 description:
                     - Key algorithm.
```

Comment:
```suggestion
                    - Key algorithm.
                    - The values V(RS384), V(RS512), ..., V(RSA-OAEP-256) have been added in community.general 8.2.0.
```
(add the remaining ones in there)